Simplenote/NoteEditorViewController.m

```diff
@@ -599,4 +581,50 @@ - (void)insertChecklistAction:(id)sender
     }
 }
 
+#pragma mark - New Note
+
+- (void)duplicateNote:(nullable Note *)sender
+{
+    if (sender != nil) {
+        [self createNoteWasPressed:sender];
+    } else {
+        [self createNoteWasPressed:self.note];
+    }
+}
+
+- (void) createNoteWasPressed:(nullable Note *)sender {
```

Comment:
Since this API isn't wired as an IBAction (and `sender` is used usually as the parameter name for the component that's doing the invocation), and UI components won't ever invoke this directly, WDYT if this was renamed as...

- `createNoteFromNote:(nullable Note *)oldNote`
- `createNoteWithNote:(nullable Note *)oldNote`
- `createNewNoteWithContent:(nullable Note *)content`
- `buildNewNoteWithContent:(nullable Note *)content`
- [Another cooler alternative here]

Thank you!

Reply:
I like your alternatives. For sure I'm will change it :)

Reply:
Thank you!